### PR TITLE
1 iter refactor rep already exists

### PR DESCRIPTION
**Pull request by pmpm02**

### CODE REVIEW

File: app/controllers/search_controller.rb [hunk 1]
```diff
@@ -9,7 +9,6 @@ def search
     service.key = Rails.application.credentials[:GOOGLE_API_KEY]
     result = service.representative_info_by_address(address: address)
     @representatives = Representative.civic_api_to_representative_params(result)
-
```
Comment (avinashrao2002) [inline app/controllers/search_controller.rb:12]: Not needed but ok to leave

File: app/models/representative.rb [hunk 2]
```diff
@@ -17,6 +17,9 @@ def self.civic_api_to_representative_params(rep_info)
         end
       end
 
+      existing_representative = Representative.find_by(name: official.name, ocdid: ocdid_temp)
```
Comment (avinashrao2002) [inline app/models/representative.rb:20]: Looks good

File: spec/models/representative_spec.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          OpenStruct.new(name: 'Jane Smith', title: 'Senator')
+        ],
+        offices:   [
+          OpenStruct.new(name: 'House of Representatives', division_id: 'ocdid1', official_indices: [0]),
+          OpenStruct.new(name: 'Senate', division_id: 'ocdid2', official_indices: [1])
+        ]
+      )
+    end
+
+    it 'not in the database' do
```
Comment (avinashrao2002) [inline spec/models/representative_spec.rb:21]: Nice job adding test for positive and negative cases

File: app/models/representative.rb [hunk 4]
```diff
@@ -17,6 +17,9 @@ def self.civic_api_to_representative_params(rep_info)
         end
       end
```
Comment (avinashrao2002) [inline app/models/representative.rb:19]: Not necessary in this PR but add comment describing what this does

**Review by avinashrao2002 — APPROVED** [review]
